src/chart.zoom.js

```diff
@@ -65,44 +65,22 @@ function rangeMinLimiter(zoomPanOptions, newMin) {
 }
 
 function zoomIndexScale(scale, zoom, center, zoomOptions) {
+	var min = scale.minIndex;
+	var max = scale.maxIndex;
+	var range = max - min;
+	var newDiff = range * (zoom - 1);
+
+	var cursorPixel = scale.isHorizontal() ? center.x : center.y;
+	var min_percent = (scale.getValueForPixel(cursorPixel) - min) / range;
```

Comment:
Please use `camelCase` for variable names. After you rebase, run `gulp lint` and it will check for style errors. You can only run `gulp lint` after rebasing since it is a recent addition